packages/backend/src/controllers/coderController.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    ApiChartAsCodeListResponse,
+    ApiChartAsCodeUpsertResponse,
+    ApiErrorPayload,
+    ChartAsCode,
+} from '@lightdash/common';
+import {
+    Body,
+    Get,
+    Hidden,
+    Middlewares,
+    OperationId,
+    Path,
+    Post,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+    Tags,
+} from '@tsoa/runtime';
+import express from 'express';
+import { allowApiKeyAuthentication, isAuthenticated } from './authentication';
+import { BaseController } from './baseController';
+
+@Route('/api/v1/projects/{projectUuid}/coder')
```

Comment:
This makes more sense attached to the existing routes in the project controller

```
GET /api/v1/projects/{projectUuid}/charts <- already exists

GET /api/v1/projects/{projectUuid}/charts/code <- to get all charts as code

POST /api/v1/projects/{projectUuid}/charts/{slug)/code <- consistent

Reply:
Done, removed this controller and moved to projectController